lib/sessions.ts

```diff
@@ -85,6 +109,98 @@ export class Sessions {
     });
   }
 
+  /** Parse a JWT and verify the signature, preferring local verification over remote.
+   *
+   * If maxTokenAge is set, remote verification will be forced if the JWT was issued at (based on
+   * the "iat" claim) more than maxTokenAge seconds ago.
+   */
+  async authenticate_jwt(
+    jwt: string,
+    options?: {
+      maxTokenAge?: number; // seconds
+    }
+  ): Promise<AuthenticateResponse> {
+    try {
+      const session = await this.authenticate_jwt_local(jwt, options);
+      return {
+        session,
+        session_jwt: jwt,
+        session_token: "",
+
+        // Pretend that this was a successful API request.
+        request_id: "",
```

Comment:
`request_id` here as well.

Reply:
This one is going away (see Max's thread above), but this was part of the "pretend that this was a successful API request". If the JWT validated locally, we wouldn't have gone to the API at all, so there wouldn't have been an API request ID to overwrite.